formValidation.js

```diff
@@ -0,0 +1,14 @@
+const email = document.getElementById('email');
+const form = document.querySelector('form');
+const errorMsg = document.getElementById('error-msg');
+
+form.addEventListener('submit', (event) => {
+  if (email.value.trim().toLowerCase() !== email.value.trim()) {
+    event.preventDefault();
+    errorMsg.className = 'error';
+    errorMsg.innerText = 'Please enter email in lowercase format!';
```

Comment:
- [ ] You should check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: email === email.toLowerCase(), utilizing regular expressions or looping through each letter of the email string. 
But this requirement is not met. I can easily submit form without an error.